File: leetcode_solutions/1732-find-the-highest-altitude/1732-find-the-highest-altitude.cs
public class Solution {
    public int LargestAltitude(int[] gain) {
        int max_altitude = int.MinValue;
        int started_trip = 0 + gain[0];
        max_altitude = Math.Max(started_trip, 0);

        for (int i = 1; i < gain.Length; i++)
        {
            int alt = started_trip + gain[i];
            started_trip = alt;
            max_altitude = Math.Max(max_altitude, alt);
        }

        return max_altitude;
        
    }
}

File: leetcode_solutions/1732-find-the-highest-altitude/1732-find-the-highest-altitude.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        max_altitude = float("-inf")
        started_trip = 0 + gain[0]
        max_altitude = max(started_trip, 0)

        for i in range(1, len(gain)):
            alt = started_trip + gain[i]
            started_trip = alt
            max_altitude = max(max_altitude, alt)

        return max_altitude
        